src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -7,6 +7,11 @@
         <fields>npe03__Installment_Period__c</fields>
         <label>NPSP Recurring Donation Compact Layout</label>
     </compactLayouts>
+    <fieldSets>
+        <description>Fields to be displayed on Recurring Donations Entry Form Custom Fields Section</description>
```

Comment:
Maybe just for clarity in case any RD1 users are wondering about this field set:
```suggestion
        <description>Fields to be displayed on Enhanced Recurring Donations Entry Form in the Custom Fields section</description>
```